### PR TITLE
Ft/green nav

### PR DESCRIPTION
**Pull request by Kevin-gram**
1. Worked on the green navbar and
2. Made it responsive on both screens. Check it out and give me your input

### CODE REVIEW

**Review by Gasorekibo — APPROVED** [review]
Everything is fine.

File: tailwind.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
       },
       colors: {
         primaryGreen: "#00672E",
-        bgPrimary: "#EFECEA",
+        primaryColor: "#EFECEA",
```
Comment (peninah98) [inline tailwind.config.js]: Can you name this customized color well?

Comment (peninah98) [reply tailwind.config.js]: It is hard to know what primaryColor mean.

File: src/sections/greenNav.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    />
+    <p
+      class="text-smallSize absolute left-4 top-2 font-semibold md:relative md:text-5xl lg:text-5xl"
+    >
+      Baumeister
+    </p>
+  </div>
+  <div class="hidden md:block">
+    <div class="flex flex-col text-2xl">
+      <a href="#about">About</a>
```
Comment (peninah98) [inline src/sections/greenNav.html:22]: You can leave it by now because you will need to navigate to the corresponding sections later

**Pull request closed without merging**